packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.html

```diff
@@ -1,4 +1,35 @@
-<section class="sf-mega-menu">
-  <!-- @slot -->
-  <slot/>
-</section>
+<div class="sf-mega-menu">
+    <section
+      class="sf-mega-menu__section"
+      :class="{'sf-mega-menu__section--is-active': active}">
+      <div class="sf-mega-menu__sidebar">
+        <SfList class="sf-mega-menu__headers">
+          <SfList
+            v-for="column in columns"
+            :key="column"
+            class="sf-mega-menu__list-item">
+            <!-- @slot Custom menu-item markup -->
+            <slot
+              name="menu-item"
+              v-bind="{updateColumn, column, active}">
+              <span @click="updateColumn(column)">
+                <SfMenuItem
+                  :label="column"
+                  class="sf-mega-menu__menu-item"
+                  :class="{'sf-mega-menu__menu-item--is-active': column === active}"
+                />
+              </span>
+            </slot>
+          </SfList>
+        </SfList>
+      </div>
+      <div class="sf-mega-menu__sidebar-content">
+        <!-- @slot Slot for other content -->
+        <slot name="menu-content"/>
```

Comment:
```suggestion
        <slot/>
```
I think is better choice for default slot :)